packages/retail-ui-extensions/src/extension-api/connectivity-api/connectivity-api.ts

```diff
@@ -0,0 +1,25 @@
+import type {RemoteSubscribable} from '@remote-ui/async-subscription';
+
+export type ConnectivityStateSeverity = 'Connected' | 'Disconnected';
+
+export interface ConnectivityState {
+  /**
+   * Whether the device is connected to the internet
+   */
+  internetConnected: ConnectivityStateSeverity;
```

Comment:
I'm curious where the Severity word came from? Sounds a little weird to me but if there's some convention, then I'm fine with it.